### ISSUE TITLE
Replacing icons in the notifications

### ISSUE THREAD

**Issue by png2378** [vikoadi/indicator-kdeconnect#22]
I use elementary OS Freya and icons in the notifications look terrible.

![Imgur](http://i.imgur.com/12bACAI.png)

![Imgur](http://i.imgur.com/PfFY7P3.png)

Used "oxygen" theme from KDE4 and it would be nice to replace it with a more modern and stylish.

**Comment by x8BitRain** [comment]
Yeah on Mint the icons are very stretched and ugly, can you atleast provide us with the names or locations of the icons so we can change them ourselves?
